components/ReportsLists/index.js

```diff
@@ -180,6 +185,25 @@ export default function Forms({ newWorkReports }) {
                     }
                   })}
               </StyledSection>
+              <StyledHeadlineThree>Arbeitsbeschreibung</StyledHeadlineThree>
+              <StyledSection>
+                {Object.entries(workReport)
```

Comment:
if someone does not type anything to `Arbeitsbeschreibung` this `StyledSection` stays empty. You should prevent this either in your form (make it required), or you put in a conditional replacement text in here, like: `---Keine Arbeitsbeschreibung---`

Reply:
Good idea! I corrected it